doc/nrf/working_with_nrf/nrf52/developing.rst

```diff
@@ -38,13 +38,14 @@ To enable support for FOTA updates, do the following:
 
 * Use MCUboot as the upgradable bootloader (:kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` must be enabled).
   For more information, go to the :doc:`mcuboot:index-ncs` page.
-* Enable the mcumgr module that handles the transport protocol over Bluetooth® Low Energy as follows:
+* Enable :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option.
 
-  a. Enable the Kconfig options :kconfig:option:`CONFIG_MCUMGR_CMD_OS_MGMT`, :kconfig:option:`CONFIG_MCUMGR_CMD_IMG_MGMT`, and :kconfig:option:`CONFIG_MCUMGR_SMP_BT`.
-  #. Call the functions :c:func:`os_mgmt_register_group()` and :c:func:`img_mgmt_register_group()` in your application.
-  #. Call the :c:func:`smp_bt_register()` function in your application to initialize the mcumgr Bluetooth Low Energy transport.
+   * By default, :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` option is implied when FOTA is enabled using mentioned option.
+     This option automatically extends the Bluetooth buffers, which allows to speed up the FOTA transfer over Bluetooth, but can also increase RAM usage.
+     You can disable this option manually if necessary.
 
```

Comment:
It would be great to add a section on how to build for a specific project like the Peripheral LBS sample. Something along these lines: 

_You can enable the FOTA feature in a single command (...) Please see how it can be done for the Peripheral LBS sample (replace the `<target>` with your target board):_

```
west build -b <target> -- -DCONFIG_BOOTLOADER_MCUBOOT=y -DCONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU=y
```